index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- link for CSS -->
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <!-- tab title -->
+    <title>Horiseon | Marketing Agency</title>
+  </head>
+
+  <body>
+    <!-- header -->
+    <header>
+      <!-- company logo -->
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <nav>
+        <ul>
+          <!-- link 1 -->
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <!-- link 2 -->
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <!-- link 3 -->
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- hero image -->
+    <div
+      class="hero"
+      alt="Professionals looking at a document in a meeting"
+    ></div>
+    <!-- services section -->
+    <main class="services">
+      <!-- services block 1 -->
+      <section id="search-engine-optimization" class="services-content">
+        <figure class="float-left services-img">
```

Comment:
Nice work incorporating the figure + figcaption, especially since it caused you issues with the positioning of the text. It actually isn't really necessary here, and the img would have been fine on its own. The alt you have added does the accessibility job :)

Reply:
thank you- i think i over killed the semantic element- but nice knowing that!